src/main/java/frc/robot/commands/VelocityDrive.java

```diff
@@ -46,9 +45,16 @@ protected void initialize() {
 
     // Called repeatedly when this Command is scheduled to run
     @Override
-    protected void execute() {
-        Robot.drivetrain.velocityDrive(Robot.oi.leftJoy.getY(), Robot.oi.rightJoy.getY(),
-         (Robot.oi.leftJoy.getTrigger() || Robot.oi.rightJoy.getTrigger()),Robot.oi.getleftJoy().getThrottle());
+    protected void execute() {;
+        boolean useSlowModifier = (Robot.oi.leftJoy.getTrigger() || Robot.oi.rightJoy.getTrigger());
+        boolean useReverse;
+        if (Robot.oi.leftJoy.getThrottle() < 0){
+            useReverse = true;
+        }
+        else{
+            useReverse = false;
+        }
+        Robot.drivetrain.velocityDrive(Robot.oi.leftJoy.getY(), Robot.oi.rightJoy.getY(),useSlowModifier,useReverse);
```

Comment:
@BobTheLegend is the idea to use the throttle so that the driver doesn't need to hold down a trigger? It feels like that would be a lot of throttle flipping and removing the driver's hand from the joystick.

Maybe assign the mode when the driver needs to use less for when the trigger is pulled?

Another alternative is to

Reply:
Yes, correct, the idea for the throttle is so that the driver will not have to hold down the trigger. Also, Jimmy tends to use both triggers to slow down and I don't want to confuse him. Eventually, I would like to switch reversing control from the throttle to a more obvious switch and maybe switch control to the operator.

Reply:
I think Jimmy would learn quick not to use both triggers. Why don't we try to use the triggers? For now I think this is good so I'm going to mark it as closed. Nice work!